Assets/XML/Text/Civilopedia.xml

```diff
@@ -3557,8 +3553,7 @@ For religions, two or three percentages are calculated:
 [ICON_BULLET]the percentage of cities that have at least one non-state religion (heathen ratio)
 [ICON_BULLET]if you are running Theocracy, the percentage of cities that have ONLY your state religion
 [NEWLINE]Some non-state religions are ignored for the purposes of the second and third ratios ("tolerance").
-[ICON_BULLET]civs with Secularism tolerate all non-state religions
-[ICON_BULLET]Mughals tolerate all non-state religions because of their UP
```

Comment:
The Mughal UP is still in effect.

Reply:
You mean the old Mughal UP of religious tolerance? There's a new Mughal UP (earning culture points from building production) since the old one became accessible to other civs as the Tolerance civic.

Reply:
Oh right ... I should be better at remembering my changes.